.gitignore

```diff
@@ -21,7 +21,7 @@
 .vscode/*

```

Comment:
this code should have ignored all other files inside .vscode so it should not have been required to add extra lines.